.circleci/config.yml

```diff
@@ -39,14 +39,14 @@ jobs:
             . venv/bin/activate
             cd doc
             git submodule update --init
-            make html
+            make html -halt-on-error
```

Comment:
This is being interpreted as flags to `make`, `-h -a -l ...`

Reply:
fixed